addons/mail/static/src/new/web/activity/activity_menu.js

```diff
@@ -40,6 +40,15 @@ export class ActivityMenu extends Component {
         this.fetchSystrayActivities();
     }
 
+    get availableView() {
```

Comment:
Should be a method that takes `group.model` as param to allow override condition.